src/api/ingest/benthicpqt_serializer.py

```diff
@@ -0,0 +1,109 @@
+from rest_framework import serializers
+
+from ..fields import LazyChoiceField, PositiveIntegerField
+from ..models import BenthicAttribute, GrowthForm, ReefSlope
+from .choices import (
+    build_choices,
+    current_choices,
+    relative_depth_choices,
+    tide_choices,
+    visibility_choices,
+)
+from .serializers import CollectRecordCSVSerializer
+
+__all__ = ["BenthicPhotoQTCSVSerializer"]
+
+
+def growth_form_choices():
+    return build_choices(GrowthForm.objects.choices(order_by="name"))
+
+
+def reef_slopes_choices():
+    return build_choices(ReefSlope.objects.choices(order_by="name"))
+
+
+def benthic_attributes_choices():
+    return [
```

Comment:
Can't this use `build_choices` too?

Reply:
build_choices expects the choices to be a dictionary not a model instance